requestmanager/asyncloader/unverifiedblockstore/unverifiedblockstore.go

```diff
@@ -30,21 +33,26 @@ func New(storer ipld.Storer) *UnverifiedBlockStore {
 // comes in as part of a traversal.
 func (ubs *UnverifiedBlockStore) AddUnverifiedBlock(lnk ipld.Link, data []byte) {
 	ubs.inMemoryBlocks[lnk] = data
+	ubs.dataSize = ubs.dataSize + uint64(len(data))
+	log.Debugw("added in-memory block", "data-size", ubs.dataSize)
```

Comment:
`data-size` is not very descriptive, and it lacks a unit.
```suggestion
	log.Debugw("added in-memory block", "total_queued_bytes", ubs.dataSize)
```